x-pack/test/plugin_functional/test_suites/resolver/index.ts

```diff
@@ -4,24 +4,174 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import expect from '@kbn/expect';
+import { WebElementWrapper } from '../../../../../test/functional/services/lib/web_element_wrapper';
+
 import { FtrProviderContext } from '../../ftr_provider_context';
 
-export default function ({ getPageObjects, getService }: FtrProviderContext) {
+const expectedDifference = 0.09;
```

Comment:
I increased this number until tests passed. One thing that I noticed: running the tests with my laptop had different results depending on whether my external monitor was connected.